.github/workflow-templates/.editorconfig

```diff
@@ -0,0 +1,23 @@
+# This file is for unifying the coding style for different editors and IDEs
+# editorconfig.org
+
+# WordPress Coding Standards
+# https://developer.wordpress.org/coding-standards/wordpress-coding-standards/
```

Comment:
Please add a comment like this so it's clear that the file is managed from outside the repos: https://github.com/gocodebox/.github/blob/a38fa0c94402895bd12af96b6cfb7dfb27a4c08e/.github/workflow-templates/coding-standards.yml#L1-L11